lib/membrane_rtc_engine/engine.ex

```diff
@@ -284,17 +264,17 @@ defmodule Membrane.RTC.Engine do
     dispatch({:new_peer, peer_id, data.metadata})
 
     receive do
-      {:accept_new_peer, ^peer_id} ->
-        do_accept_new_peer(peer_id, node(), data, ctx, state)
+      {:accept_new_peer, ^peer_id, bin} ->
```

Comment:
Let's rename `bin` to `endpoint`. Please check other places to (e.g. `do_setup_new_peer`)